### ISSUE TITLE
hang on Bio update

### ISSUE THREAD

**Issue by dominix** [weskerfoot/DeleteFB#56]
I have a Bio update can can not be deleted, so the script hang on the page, waiting for nothing and can not go further.

**Comment by weskerfoot** [comment]
@dominix Does it give any option to hide the update?

**Comment by weskerfoot** [comment]
I think it might need some functionality to handle these types of cases where it scrolls down and skips posts that cannot be deleted or hidden, but I'm unsure of how to implement that right now.

**Comment by weskerfoot** [comment]
I'm not sure if this is an issue for newer accounts. I think if you have really old posts (going back over a decade), then it comes up a lot more often as they've changed the site a lot over the years.

**Comment by dominix** [comment]
your're right, update comes from 2011. account is even older.

**Comment by weskerfoot** [comment]
Yeah, I would like to have a way to gracefully deal with those types of posts. I'm not sure whether there's a way to skip them or not.
